tardis/plasma/tests/test_property_ion_population.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+from tardis.plasma.properties.ion_population import (PhiSahaLTE,
+IonNumberDensity)
+from tardis.plasma.properties.general import (BetaRadiation, GElectron,
+NumberDensity)
+from tardis.plasma.properties.partition_function import (LevelBoltzmannFactor,
+LTEPartitionFunction)
+from tardis.plasma.properties.atomic import Levels, AtomicMass, IonizationData
+
+def test_phi_saha_lte(t_rad, included_he_atomic_data):
+    beta_rad_module = BetaRadiation(None)
```

Comment:
yep. you also define fixtures in the `conftest.py` in the directory above. then they will be available for all functions.